src/routes/appointments.js

```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router();
+
+const { create } = require('../controllers/appointments');

```

Comment:
Create what? I think need more clarifying name vars.

Reply:
Rename it to `createAppointment`